### ISSUE TITLE
Individual form templates aren't used

### ISSUE THREAD

**Issue by marck-consulting** [bobdenotter/forms#11]
If bolt-boltforms.yaml template block is configured as:
```
templates:
    form: 'extensions/boltforms/form.html.twig'
    email: 'extensions/boltforms/email.html.twig'
    subject: 'extensions/boltforms/subject.html.twig'
    files: 'extensions/boltforms/file_browser.twig'
```

and a form is configured as:
```
## Subscribe
subscribe:
    templates:
        form: 'partials/_subscribe.twig'
    feedback:
        success: Deine E-Mail Adresse email wurden erfolgreich eingetragen.
        error: Die eingegebene E-Mail Adresse ist ungültig.
    fields:
        email:
            type: email
            options:
                required: true
                label: E-Mail Adresse
                attr:
                    placeholder: E-Mail Adresse
                constraints: [ NotBlank, Email ]
        submit:
            type: submit
            options:
                label: Submit
                attr:
                    class: button primary
```
The used template is still "form.html.twig" when it should be "_subscribe.twig" instead.

**Comment by iamgoodbytes** [comment]
This issue is related to this: https://github.com/bolt/forms/issues/19

**Comment by yammycode** [comment]
Even the global setting does not work for me.
1. I created a folder in the directory:  public /theme/ my_theme/extensions /boltforms /.
2. Copied email.html.twig into it.
3. Changed the markup in this file.
4. In config/extensions/bolt-boltforms.yaml set:
templates:
    form: '@boltforms/form.html.twig'
    email: 'extensions/boltforms/email.html.twig'
    subject: '@boltforms/subject.html.twig'
    files: '@boltforms/file_browser.twig'

But messages are sent anyway according to the default template ... What am I doing wrong?

**Comment by benr77** [comment]
I can confirm this - Bolt 4 default install - the form specific template directives are ignored and only the global ones accounted for

**Comment by benr77** [comment]
This is still not fixed in 1.0.8

**Comment by I-Valchev** [comment]
I had the same issue, but I think part of the problem was the twig path thing.

What you can do:

1. Edit `config/packages/twig.yaml`  by uncommenting this line:

https://github.com/bolt/project/blob/1b2e8b9d41759748e56bfafc6daaa212bcd461fe/config/packages/twig.yaml#L15

Then, in the forms config, you can do this:

```
templates:
    email: '@theme/forms/email.html.twig'
    # ...
```

Which will look for a `forms` folder in your theme, inside which it expects `email.html.twig`

**Comment by benr77** [comment]
This is the setup I have already, using `@theme` etc - it finds the template correctly with this notation, but only if I use it in the `templates` block in the global scope. If I try and use `templates` in a form specific context, it is ignored.

**Comment by I-Valchev** [comment]
Okay, I see! I didn't need to do this yet, so haven't tested it. Let's re-open the issue for individual templates in that case :-)

It is probably the case that it's not implemented.

**Comment by kryst3q** [comment]
This should fix the issue: https://github.com/bolt/forms/pull/49

**Comment by bobdenotter** [comment]
Fixed in #49
